src/main/java/com/plaid/client/request/ApexProcessorTokenCreateRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.plaid.client.request;
+
+import com.plaid.client.request.common.BaseAccessTokenRequest;
+
+/**
+ * Request for /item/public_token/exchange endpoint.
+ */
+public final class ApexProcessorTokenCreateRequest extends BaseAccessTokenRequest {
+  private String accessToken;
```

Comment:
`BaseAccessTokenRequest` already defines `accessToken`, so this line should be removed - it's not used anywhere.